src/components/common/Childnav.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
```

Comment:
this is a new componant, housing all the code from the Header.js file. Header.js now just holds a function that renders headers based off the current URL pathname.